backend/app.js

```diff
@@ -99,7 +99,7 @@ app.use('/api/solr', solrRouter);
 app.use('/api/resource', resourceRouter);
 app.use('/api/ckan', ckanRouter);
 app.use('/api/pow', powRouter);
-app.use('/api/analytics', analyticsRouter);
+app.use('/api/an', analyticsRouter);
```

Comment:
Was there a reason for changing this? (an is not a very clear naming)

Reply:
If you leave it as analytics it gets blocked by a specific analytics blocker (uBlock Origin)